elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```diff
@@ -142,8 +142,9 @@ protected void configure() {
 
                     // TODO: If null, initialize with FileResultStorageEngine
                     ResultStorageEngine resultStorageEngine = asyncProperties.getResultStorageEngine();
+                    // TODO: Add logic to generate baseURLPresent Flag
```

Comment:
Will do separate PR for Standalone to set baseURLPresent flag and binding of Table Export Hooks.